### ISSUE TITLE
the difference between meso and mesoD

### ISSUE THREAD

**Issue by ZERO6666** [alright1117/Decoy#2]
in your code, i didn't see any difference between the meso network and the mesoD network;
both of them are

	def __init__(self):
		super(MesoDeception, self).__init__()
		self.model1 = MesoInception4(num_classes=1)
		self.model2 = MesoInception4(num_classes=1)
	
	def forward(self, input):
		real_score = self.model1(input)
		fake_score = self.model2(input)
		
		return torch.cat((real_score, fake_score), axis = 1)

is that right?
